Assets/Scripts/Enemy/Enemy States/EnemyAttack.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace BattleTank
```

Comment:
Please complete this code or if you have other alternative then delete this code